packaging/msi/Product.wxs

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix
+    xmlns="http://wixtoolset.org/schemas/v4/wxs"
+    xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui"
+    xmlns:util="http://wixtoolset.org/schemas/v4/wxs/util">
+    <Package
+        Name="Ice 3.8.0"
+        Manufacturer="ZeroC, Inc."
+        Version="3.8.0.0"
+        InstallerVersion="500"
+        Compressed="yes"
+        UpgradeCode="DDC28522-EF11-4D6C-9F53-B76DF967C8B2">
+
+        <!-- Embed CAB files in MSI -->
+        <MediaTemplate EmbedCab="yes" />
+
+        <!-- Define the product icon -->
+        <Icon Id="icon.ico" SourceFile="!(bindpath.SOURCE_ROOT)\packaging\msi\resources\product.ico"/>
+        <Property Id="ARPPRODUCTICON" Value="icon.ico" />
+
+        <!-- Use the default WixUI_InstallDir dialog set and personalize the license an images -->
```

Comment:
This built-in dialog set is almost identical to what we have in 3.7.10, with the exception of the "Add bin directory to the system PATH environment variable" in the install folder dialog.